pusher.go

```diff
@@ -0,0 +1,298 @@
+package pusher
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/pusher/pusher-http-go/authentications"
+	"github.com/pusher/pusher-http-go/errors"
+	"github.com/pusher/pusher-http-go/requests"
+	"github.com/pusher/pusher-http-go/signatures"
+	"github.com/pusher/pusher-http-go/validate"
+	"net/http"
+	"net/url"
+	"time"
+)
+
+const (
+	NATIVE_NOTIFICATIONS_HOST = "yolo.ngrok.io"
```

Comment:
@jpatel531 it took me some time to realize I was making calls to your localhost :) Shouldn't this be configurable in `Options`?